### DISCUSSION TITLE
DsHidMini Driver v2.2.282

### DISCUSSION THREAD

**Discussion by nefarius** [nefarius/DsHidMini#159] [Announcements]
# v2.2.282 DsHidMini changelog

This project was only possible with the investment of many **years** of learning, research and dedication. You're receiving this work for free, if it brought you joy, consider giving back and spend a few coins ❤️

[![DonateEUR](https://user-images.githubusercontent.com/286631/126007336-0b0cfa6d-b0bd-408f-9c1f-46c4227778fb.gif)](https://vigem.org/Donations/)

## How to install/update/uninstall

[Follow the this comprehensive guide](https://vigem.org/projects/DsHidMini/How-to-Install/).

## How to use in emulators and games

**You** need to configure the driver first for best experience!

Carefully read the provided documentation to configure the driver for best game compatibility. 

99% of your questions are already answered 😉 

👉 [About DsHidMini](https://vigem.org/projects/DsHidMini/)
👉 [About modern Games compatibility](https://vigem.org/projects/DsHidMini/XInput-Mode-Explained/)
👉 [Info regarding using your DS3 as a Xbox/DS4 controller with DsHidMini](https://github.com/ViGEm/DsHidMini/issues/40)
👉 [DsHidMini's DS4 Mode user guide](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)

## Remarks

⚠️ **This is a new major release which breaks backwards compatibility!** ⚠️

This release **is not compatible with** [BthPS3](https://github.com/ViGEm/BthPS3) version **1**.x.x! An **upgrade** to the latest **2**.x.x release is mandatory!

## Features

- Made DSHMC window resizable for systems with DPI scaling issues (#157)
- Added warning dialog when applying device settings didn't work (#158)
- Added SCP-compatible Extended XInput Proxy DLL (#66)

<hr /><em>This discussion was created from the release <a href='https://github.com/ViGEm/DsHidMini/releases/tag/v2.2.282.0'>DsHidMini Driver v2.2.282</a>.</em>

**Comment by Speedflux** [comment]
Hey, I am unable to install the two .inf file on my PC (Windows 11). I right-clicked "Install", nothing, I used command prompt, it says installation failed, 

I also tried with the immediate previous version, same issh.
What is the solution to this problem?

**Reply by guoqingkong** [reply]
You can use the device management 's add the driver menu in the bar.  But I get the diffirent problem, the driver said error 19..........
![2022-11-17](https://user-images.githubusercontent.com/320993/202434494-bea5e866-f91d-46d3-aeda-fa41007565b0.png)

**Comment by vrmvgf** [comment]
I played Hollow Knight with DS3 controller In Win10 Xbox PC App,  the controller disconnected about every ten minutes, sometimes it can reconnect later.

Is there something wrong?

**Reply by nefarius** [reply]
IDK, is it charged?

**Reply by vrmvgf** [reply]
Yes, I used USB cable while playing

**Reply by pftmclub** [reply]
i got the same problem. Even with wireless mode !

**Comment by OneNobody** [comment]
Malwarebytes  Malware.Heuristic.1001 detected

**Reply by nefarius** [reply]
Report a false-positive to Malwarebytes then 😛

**Reply by OneNobody** [reply]
> Report a false-positive to Malwarebytes then 😛


So do I understand this correctly, that I don't have to open an electronics store after installing the driver, because new devices will be delivered to me for weeks? Thanks. :D :P

**Reply by nefarius** [reply]
No idea what you're talking about.

**Reply by OneNobody** [reply]
It was a joke... what ever... Now I get this message: "This hardware device cannot be started because its configuration information (in the registry) is incomplete or corrupted. (Code 19)"

**Reply by nefarius** [reply]
Too bad 😬

**Comment by k-kurt** [comment]
596 / 5.000
Resultados de traducción
Does anyone know how to get the controller back on the ps3? because after having used it on the pc with this program I can't use it on the ps3, the PS button doesn't work, the issue is that I use the controller on the pc because the ps3 was blocked due to an update and now you can go back to unlock the ps3 so when I wanted to use the controller the ps button doesn't work but when I connect the controller to the pc with the ps3 plugged in it turns on the ps3, as the program took the PS button from the controller and I don't know how to make it return operation of the PS button to normal, I already reset the remote and nothing

**Reply by nefarius** [reply]
You plug it back into a PS3, done.

**Reply by k-kurt** [reply]
obviously I already did it and it doesn't work, for me it's not that the button was broken because when I connect it to the PC and the PC turns on, the play also turns on because the button is taken by the program and then I go and I connect to the ps3 and the PS button does not work but the other buttons do but for me the button is taken

**Reply by nefarius** [reply]
Well, that is the only procedure, beside trying the reset hole on the back as per FAQ.

**Reply by Kanuan** [reply]
When you connect the controller to the PS3 via cable  the console pairs the controller to itself. It's impossible for dshidmini to prevent or break this function, specially if you reset the controller using the button on its back.

If you are connecting the controller via cable to the console and this is not working then probably the cable you are using is faulty: it's working as a charging cable but is not a transmitting data, so the console can't communicate with the DS3.

Another possibility is that the cable is fine but either the console or the controller USB port is faulty and is not communicating data properly.

**Reply by k-kurt** [reply]
ok, thank you very much for your answers

**Comment by L0r3mIpsum** [comment]
Does this BT method makes the BT useless like when the ScpToolkit overrited the driver to only use it with the PS3 controller or I can use it alongside the rest of my BT devices?

**Reply by nefarius** [reply]
The latter.

**Comment by Blitz54** [comment]
Mostly wanted to say thank you so much for this. I bought a PS2 to USB adapter, it worked just plug and play, but the vibration didn't work, The adapter (a black single PS2 port) was meant to work with PS3 as well, and came up as a PS3 controller in the device manager. After some searching I came across this. Installed the drivers, and BAM the vibration worked on my PS2 controller. So I'm happy now, not sure where else I could have said thanks so hopefully this is an appropriate enough location.

**Reply by nefarius** [reply]
Giving kind words is appropriate in *any* location 😉🙏

**Comment by KerrmexRPG** [comment]
Hi Nefarius, thanks for this amazing tool! I wanna ask you, is this compatible with Windows 11?

**Reply by nefarius** [reply]
Yes.

**Comment by xXPhantom-JXx** [comment]
Hey this might be a stupid question to ask, but do I need to download all 3 assets or just the first one?

**Reply by nefarius** [reply]
The first unless you know what to do with the sources 😉

**Comment by dDulee** [comment]
Hi! I have a small issue when trying to connect 1 of 2 DS3 controllers. Both work over USB, however only one will connect via bluetooth. I can't figure out what's wrong. I have a TP-Link UB500 adapter which is 5.0 bluetooth capability and a CSR 4.0 adapter. I only had one connected while installing, and even if i connect both at the same time, or only one and add another one or disconnect one, the Bluetooth connection is not interrupted at all. But, i simply can't figure out why the other DS3 won't connect over bluetooth. Do you have any way to specify which controller is going to use which adapter? I'm running out of options here, thanks!

**Comment by Benedict1610** [comment]
My system doesn't have a Bluetooth Port driver installed and i've tried installing different drivers but they don't even install.....do you think i can still run this program?

**Reply by nefarius** [reply]
As per documentation, DsHidMini **wired only** has no direct dependency to Bluetooth. Anything else you need to figure out for your particular system on your own by experimentation.

**Comment by AlfaTTM** [comment]
Hola, nescesito ayuda, cuando voy a ejecutar la instalación para poder usar el mando de ps3, me aparece este error
![Captura de pantalla (17)](https://github.com/ViGEm/DsHidMini/assets/131791852/db9f5132-d98b-40d8-b746-f4072f2ade85)
¿Que debo hacer? Ya desinstale los drivers pero me aparece igual

**Reply by nefarius** [reply]
[Read the FAQ](https://vigem.org/projects/BthPS3/Frequently-Asked-Questions/).
